stable/weave-scope/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: weave-scope
-version: 1.1.5
+version: 1.1.6
```

Comment:
It would be nice if the `version` patch number followed `appVersion`'s. Luckily, Scope got a [new release](https://github.com/weaveworks/scope/releases) in the meantime, so maybe the easiest thing to do would be to bump the `appVersion` to `1.11.6` in this PR :)

Reply:
Given that chart changes can happen while the application version stays the same (e.g. due to breaking changes in Kubernetes), it is not likely this will be aligned for a long time.

But given

> Thinking about it, it might be a hassle to maintain the patch version match in the future, but it would be good bump to the new release of Scope in this PR while you're at it :)

you probably reached the same conclusion.